routes/root/index.js

```diff
@@ -3,38 +3,141 @@
+     * @param {number} perPage events per page
+     * @param {number} page page number (1 based)
+     * @param {moment} start start date
+     * @param {moment} end end date
+     * @param {string} defaultImage default image
+     * @returns {Promise} promise for events
+     */
+    async function listEvents(perPage, page, start, end, defaultImage) {
+      const eventsApi = Common.getLinkedEventsEventsApi(config);
+
+      const listOptions = {
+        sort: "end_time",
+        page: page,
+        pageSize: perPage,
+        include: ["location"]
+      };
+
+      if (start) {
+        listOptions["start"] = start.toDate();
+      }
+
+      if (end) {
+        listOptions["end"] = end.toDate();
+      }
+
+      const events = (await eventsApi.eventList(listOptions)).data;
+      return events.map(event => translateEvent(event, defaultImage));
+    }
+
+    /**
+     * Translates LinkedEvents event into format expected by the pug templates
+     *
+     * @param {any} event LinkedEvents event
+     * @param {string} defaultImage default image
+     * @returns {any} translated event
+     */
+    function translateEvent(event, defaultImage) {
+      const shortDescription = (event.short_description ? event.short_description.fi : "") || "";
+      const description = (event.description ? event.description.fi : "") || "";
+      const originalUrl = (event.info_url ? event.info_url.fi : "") || "";
+      const name = (event.name ? event.name.fi : "") || "";
+      const imageUrl = event.images && event.images.length ? event.images[0].url : null;
+      const place = event.location ? event.location.name.fi : "";
+      const start = formatDate(event.start_time, event.end_time);
+
+      return {
+        id: event.id,
+        name: name,
+        place: place,
+        originalUrl: originalUrl,
+        start: start,
+        shortDate: moment(event.start_time).format("D.M.YYYY"),
+        imageSrc: imageUrl || defaultImage,
+        description: Common.plainTextParagraphs(Autolinker.link(description)),
+        shortDescription: _.truncate(shortDescription, { length: 200 })
+      };
+    }
     
-    app.get('/', (req, res, next) => {
+    app.get('/', async (req, res, next) => {
+      const eventsToday = await listEvents(Common.EVENT_COUNT, 1, moment(), undefined, "/gfx/layout/tapahtuma_default_120_95.jpg");
+
       new ModulesClass(config)
-        .events.latest(Common.EVENT_COUNT, 'START_DATE', 'DESCENDING')
         .news.latest(0, 9)
         .banners.list()
         .tiles.list()
         .socialMedia.latest(Common.SOCIAL_MEDIA_POSTS)
         .jobs.list(Common.JOB_COUNT, 'PUBLICATION_END', 'ASCENDING')
         .announcements.list(Common.ANNOUNCEMENT_COUNT, 'PUBLICATION_DATE', 'DESCENDING')
+        .events.latest(Common.EVENT_COUNT, 'START_DATE', 'DESCENDING')
         .callback(function(data) {
-          var events = _.clone(data[0] || []).map(event => {
+          var events = _.clone(eventsToday || []).map(event => {
```

Comment:
This clone no longer needed because event translator already creates cloned event objects